src/services/WolfDen.Application/Requests/Queries/Attendence/AttendanceSummary/AttendanceSummaryQueryHandler.cs

```diff
@@ -53,6 +53,12 @@ public async Task<AttendanceSummaryDTO> Handle(AttendanceSummaryQuery request, C
                     break;
                 }
 
+                if (currentDate.DayOfWeek == DayOfWeek.Saturday || currentDate.DayOfWeek == DayOfWeek.Sunday)
+                {
+                    continue; 
+                }
+
+
                 DailyAttendence attendanceRecord = attendanceRecords.FirstOrDefault(x => x.Date == currentDate);
```

Comment:
check how this complexity can be reduced by declaring a dictionary instead of list